saturn-tensorflow/environment.yml

```
@@ -1,7 +1,7 @@
 name: saturn
 channels:
-- defaults
 - conda-forge
+- defaults
```

Ah good catch! I would recommend going a step further and making this 
```suggestion
- nodefaults
```

---

I tried this - something in this image (and not in others) is only available in `defaults`, and it can't solve without it.